src/cozmo/run.py

```diff
@@ -780,7 +780,8 @@ def setup_basic_logging(general_log_level=None, protocol_log_level=None,
 def run_program(f, use_viewer=False, conn_factory=conn.CozmoConnection,
                 connector=None, force_viewer_on_top=False,
                 deprecated_filter="default", use_3d_viewer=False,
-                show_viewer_controls=True):
+                show_viewer_controls=True,
+                throw_error=False):
```

Comment:
New arguments need documenting in the docstring just below this too (see the pattern for use_3d_viewer, show_viewer_controls etc.).
throw_error is a bit unclear - I think it would be clearer to name it "exit_on_connection_error" (defaulted to True), and explain in the docstring that if this is set to False then it will re-raise the exception instead.

Reply:
okey dokey. I see your point
Will do!